synapse/lib/httpapi.py

```diff
@@ -145,7 +145,7 @@ def loadJsonMesg(self, byts, validator=None):
             self.sendRestErr('SchemaViolation', 'Invalid JSON content.')
             return None
 
-    def logAuthIssue(self, mesg=None, user=None, username=None, level=logging.WARNING):
+    async def logAuthIssue(self, mesg=None, user=None, username=None, level=logging.WARNING):
```

Comment:
Why did this need to become async?

Reply:
Vestigial. Originally due to getLogExtra being async. I'll revert :+1: